src/extractor/edge_based_graph_factory.cpp

```diff
@@ -138,7 +138,7 @@ void EdgeBasedGraphFactory::InsertEdgeBasedNode(const NodeID node_u, const NodeI
     NodeID current_edge_source_coordinate_id = node_u;
 
     // traverse arrays from start and end respectively
-    for (const auto i : util::irange(0UL, geometry_size))
+    for (const auto i : util::irange(std::size_t{ 0 }, geometry_size))
```

Comment:
I think this is due to the long literal as long is 4 byte on 64bit Windows, 8 byte on 64bit Unix, but I'm unsure why the compiler complains only here.